crates/holochain_p2p/src/spawn/actor.rs

```diff
@@ -291,18 +291,6 @@ impl kitsune2_api::SpaceHandler for HolochainP2pActor {
                             tracing::debug!(?err, "Error sending call remote response");
                         }
                     }
-                    RemoteSignalEvt {
```

Comment:
just moved this down so the two events can be together

Reply:
They'll be so happy!